libafl/src/monitors/disk.rs

```diff
@@ -81,8 +81,13 @@ exec_sec = {}
             )
             .expect("Failed to write to the Toml file");
 
-            for (i, client) in self.client_stats_mut().iter_mut().enumerate() {
-                let exec_sec = client.execs_per_sec(cur_time);
+            for i in 0..(self.client_stats().len()) {
+                let client_id = ClientId(i as u32);
```

Comment:
Not sure if this is 100% correct: IIRC ClientIds might be sparse numbers, especially when clients quit and new ones join

Reply:
Possible that it's been wrong all along, yep :D

Reply:
Looking into it further, we never created `ClientId` from `i` before, so I would argue it was "more" correct.

Reply:
But the implementation of `client_stats_for` and `client_stats_for_mut` is just take the inner value of `ClientId` as index to access `client_stats` vector, and `client_stats_mut` is just returning the underlined vector in all trait implementations

Reply:
Yeah.. that's not perfect. It probably works because we never drop stale clients from the stats list (?)

Reply:
Can we.. somehow fix this? Or should I ignore it for now